include/bm_baselineGB.hpp

```diff
@@ -0,0 +1,591 @@
+//
+// Created by Adrián on 16/4/24.
+//
+#include <cstdlib>
+#include <iostream>
+#include <string>
+extern "C"
+{
+#include "../../../GraphBLAS/Include/GraphBLAS.h"
+#include "../../../LAGraph/include/LAGraph.h"
+}
+
+#ifndef RPQ_MATRIX_BASELINEGB_HPP
+#define RPQ_MATRIX_BASELINEGB_HPP
+#define fullSide (~(uint)0) 
+
+#define mmax(x, y) (((x) > (y)) ? (x) : (y))
+namespace bm_baselinegb
+{
+    extern "C"
+    {
+#include "baseline/utilstime.h"
+    }
+
+    class wrapper
+    {
+        // TODO: add error handling. and think about memory leaks
+    public:
+        static const uint64_t full_side = fullSide;
+        typedef GrB_Matrix matrix_type;   // (*** in original this is a pointer to matrix ***)
+        typedef GrB_Matrix s_matrix_type; // (*** and this is matrix itself ***)
+
+        static void time_begin()
+        {
+            time_beg();
+        }
+
+        // destroys matrix
+        static inline void destroy(GrB_Matrix M)
+        {
+            GrB_Matrix_free(&M);
+        };
```

Comment:
`GrB_Matrix_free` change M variable. But in C variable `M` is just a copy. 
So function call like `destroy(A)` don't change A variable.
I don't know, maybe it's preferred behaivor for your task

```suggestion
        static inline void destroy(GrB_Matrix *M)
        {
            GrB_Matrix_free(M);
        };
```